logos/src/logos/main.py

```
@@ -338,91 +495,184 @@ async def generalstats(data: LogosKeyModel):
+        try:
+            async for chunk in _pipeline._executor.execute_streaming(context, payload):
+                yield chunk
+                
+                # Parse chunk for logging
+                line = chunk.decode().strip()
+                if line.startswith("data: ") and line != "data: [DONE]":
+                    try:
+                        blob = json.loads(line[6:])
+                        if first_chunk is None:
+                            first_chunk = blob
+                        if "choices" in blob and blob["choices"]:
```

Ruff (0.14.7)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)